posthog/ph_client.py

```diff
@@ -1,53 +1,41 @@
-from posthog.utils import get_instance_region
-from posthog.cloud_utils import is_cloud
-from typing import Any
 import structlog
-
+from django.conf import settings
+from typing import Any
 from contextlib import contextmanager
 
+from posthog.cloud_utils import is_cloud
+from posthoganalytics.exception_capture import Integrations
+from posthog.utils import get_instance_region
+
 PH_US_API_KEY = "sTMFPsFhdP1Ssg"
 PH_US_HOST = "https://us.i.posthog.com"
 
 logger = structlog.get_logger(__name__)
 
 
-def get_ph_client():
+def get_us_client(**kwargs: Any):
     from posthoganalytics import Posthog
 
     if not is_cloud():
         return
 
-    # send EU data to EU, US data to US
-    api_key = None
-    host = None
-    region = get_instance_region()
-    if region == "EU":
-        api_key = "phc_dZ4GK1LRjhB97XozMSkEwPXx7OVANaJEwLErkY1phUF"
-        host = "https://eu.i.posthog.com"
-    elif region == "US":
-        api_key = PH_US_API_KEY
-        host = PH_US_HOST
```

Comment:
@PostHog/team-data-warehouse looks like ya'll added this as something to do with capturing events for billing purposes. @Gilbert09 mentioned you may not be using celery tasks anymore. Is it ok to send these events to the US or do they still need to go to the EU?

Reply:
Yeep - we have no need for this anymore, our billing is done by querying postgres now